### PR TITLE
Random int pr for upstream

### PR DESCRIPTION
**Pull request by ncatelli**
This commit adds a randomintrange template function for generating random integers between a range. An example usage would be

```
{{ . | randomintrange 0 5}}
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,4 +18,4 @@ before_install:
   - go get github.com/mattn/goveralls
   - if ! go get code.google.com/p/go.tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
 script:
-    - $HOME/gopath/bin/goveralls -service=travis-ci
```
Comment (leekchan) [inline .travis.yml]: Hello. Thanks for the nice PR.

BTW, It does not seem to be completely rolled back. Could you do a complete rollback?

**Comment by ncatelli** [comment]
Hello @leekchan,

Sorry about that. This has been resolved and is ready for your review.

File: .travis.yml (hunk 1 above)
Comment (ncatelli) [reply .travis.yml]: completed

**Comment by leekchan** [comment]
Cool! Thanks 👍

**Pull request merged**